docs/wiki/user/radio-signal-loss.md

```diff
@@ -61,3 +61,13 @@ API function example:
 `[true] call acre_api_fnc_ignoreAntennaDirection;`
 
 `true` enables is used to ignore the antenna direction.
+
+###  Automatic Antenna direction (only in Dev-Build)
```

Comment:
Just use `{% include note.html content="Development Build only!" %}` below the title (with one line above and below it empty), will get you a fancy note. You can also just add `introduced in v2.7.0` already.

Reply:
Check